### PR TITLE
chore(release): #8 - Set-up semantic-release

### PR DESCRIPTION
**Pull request by grzesuav**
PLEASE DO NOT MERGE YET

Things which need to be changed/setup
1. change repo name in relese-docker workflow from `grzesuav/metacontroller` to `metacontroller/metacontroller`
2. Set up NEXUS_USERNAME and NEXUS_PASSWORD (as access token, not actual password) secrets
3. Set up PAT (personal access token) secret

To check :
* release job passed - https://github.com/grzesuav/metacontroller/actions/runs/125241273 and triggered a 0.5.0 release (when I merged fake commit)
* on tag 0.5.0 - docker push action passed - https://github.com/grzesuav/metacontroller/actions/runs/125244351 and triggered this docker  image  - https://hub.docker.com/repository/docker/grzesuav/metacontroller

### CODE REVIEW

**Comment by grzesuav** [comment]
Alternatively, I can for now change trigger for release from automatic to webhook action. Then we could trigger release with postman request, on demand. We could get back to automatic releasing after triggering initial release, and adding more tests etc. What do you think ?

File: .github/workflows/release-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Deploy docker image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: grzesuav/metacontroller
```
Comment (AmitKumarDas) [inline .github/workflows/release-pipeline.yaml:19]: will this be changed to docker hub repo i.e. `metacontrollerio`
i.e. https://hub.docker.com/orgs/metacontrollerio

Comment (grzesuav) [reply .github/workflows/release-pipeline.yaml:19]: yes, I used my own to test the flow

**Pull request merged**